analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -1,15 +1,30 @@
 ﻿using System;
 using System.IO;
+using System.Data.Common;
 
-public interface IInterface1 : IDisposable { }
+public interface IWithDispose : IDisposable { }
 
 class Program
 {
+    private IDisposable disposable;
+
+    public void DisposeField()
+    {
+        disposable.Dispose();
+        disposable.Dispose(); // Noncompliant
+    }
+
+    public void DisposePotentiallyNullField(IDisposable d)
+    {
+        d?.Dispose();
+        d?.Dispose(); // FN
+    }
+
```

Comment:
The 1st case in the test file is typically the default one, with ^^^^ annotationand message. Like the one below. Please move these two methods somewhere below